nixos/modules/services/continuous-integration/hydra/default.nix

```
@@ -231,7 +231,6 @@ in
     users.users.hydra =
       { description = "Hydra";
         group = "hydra";
-        createHome = true;
         home = baseDir;
```

I'd add a comment (probably want to apply this locally and reflow) so this doesn't come back:

```suggestion
        # We don't enable `createHome` here because the creation of the home directory is handled by the hydra-init service below.
        home = baseDir;
```

Maybe even that comment (slightly reworded) + an explicit `createHome = false`? Probably unnecessary though.

---

Better yet, we should add to the NixOS manual to never use `createHome` to provision directories. It is almost **always** a bad idea.

---

@aanderse: I ended up going with the comment suggested by @cole-h for now, mainly because it's still not clear to me why using `createHome` causes issues in general

---

@Gabriel439 perfectly reasonable. @zimbatm was able to provide a few arguments against using `createHome` for service directory provisioning [here](https://github.com/NixOS/nixpkgs/pull/56265#discussion_r267018798).